### DISCUSSION TITLE
Get Node's Tree Path

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10312] [Q&A] [answered]
Hi,
Consider we have a Node that is named child, and is located within parent Node in the tree. By default does theia provide any method to get it's root path like parent/child ? If not is there any easy ways to implement the same ?

Thanks

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp I believe the question is a little high-level but `tree-nodes` have a `parent` (which you can access) if you look at the API:

- https://eclipse-theia.github.io/theia/docs/next/interfaces/core.treenode-3.html#parent

**Reply by maheshrajrp** [reply]
Thanks for your reply, @vince-fugnitto . I'll try to implement via the same.
